examples/load_and_upsample_bboxes.py

```diff
@@ -277,10 +277,7 @@ def plot_position_and_shape_xy_coords(ds_input_data, ds_filled, color_filled):
 
 for data_array_str in ["position", "shape"]:
     ds_interp[data_array_str] = interpolate_over_time(
-        data=ds_interp[data_array_str],
-        method="linear",
-        max_gap=None,
-        print_report=False,
+        data=ds_interp[data_array_str]
```

Comment:
This is a not very important, but I would leave the `method="linear"` , `max_gap=None` and `print_report=False`  inside the function call. I am aware that these are the default values and hence this is equivalent to not explicitly naming them, but I think it makes the function call more readable (i.e. the user knows exactly what the function is doing), which is important in these educational examples.

Feel free to disagree with my advice here, I don't feel particularly strongly about it.

Reply:
Thanks for the feedback! I removed the explicit parameters since they are the default values, but I understand your point about readability, especially in educational examples. I'll revert the change to keep the function call clearer for users.